kodilla-patterns2/src/main/java/com/kodilla/patterns2/adapter/bookclasifier/MedianAdapter.java

```diff
@@ -0,0 +1,25 @@
+package com.kodilla.patterns2.adapter.bookclasifier;
+
+import com.kodilla.patterns2.adapter.bookclasifier.librarya.Book;
+import com.kodilla.patterns2.adapter.bookclasifier.librarya.Clasifier;
+import com.kodilla.patterns2.adapter.bookclasifier.libraryb.BookSignature;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class MedianAdapter extends MedianAdaptee implements Clasifier {
+
+    @Override
+    public int publicationYearMedian(Set<Book> bookSet) {
+        Map<BookSignature, com.kodilla.patterns2.adapter.bookclasifier.libraryb.Book> books;
+        books = bookSet.stream()
```

Comment:
Z jakiego powodu rozbiłeś deklarację zmiennej i przypisanie wartości na 2 linie? (to nie błąd, interesuje mnie Twoja motywacja).